crates/driver/src/domain/liquidity/zeroex.rs

```diff
@@ -30,6 +31,15 @@ pub struct Order {
     pub signature: ZeroExSignature,
 }
 
+#[derive(Clone, Debug)]
+pub struct FillableAmounts {
```

Comment:
nit: Just `Amounts` so it can be reused for maker_amount/taker_amount.

Are the original maker/taker amounts on the order still relevant for something? If not, let's remove them to make it less hard for a future maintainer to not accidentally use those kind of pointless amounts.

Reply:
Renamed. Original values are still required for the settlement encoding.